### ISSUE TITLE
[Suggestion] Different icon for players when shared with them

### ISSUE THREAD

**Issue by MalakanC** [death-save/pin-cushion#59]
I am making my notes following a journal guide by Pyram King on Youtube which involves putting the room description, and any associated art, into a journal. Below this, in a 'secret' section, are my GM notes. The journals are shared to the player as 'observers' after they have entered the room. This allows them to see the room descriptor for future reference but also prevents them seeing the GM notes. A tidy solution.

I'm also using the `Automatic Journal Icon Numbers` module, which allows me to drop the journal on the map and also number the rooms for my reference. The issue I face is that the icon with a number is also shared to the players, I would like for the icon to be different for the players as to not show the room number.

tl;dr - have the ability to have gm icons separate to player icons for journals on scenes.

**Comment by p4535992** [comment]
Done on my fork https://github.com/p4535992/pin-cushion/releases/latest/download/module.json, but nobody really used it.
